packages/odyssey-storybook/.storybook/preview.js

```diff
@@ -1,5 +1,3 @@
-import "@okta/odyssey-react/dist/odyssey-deprecated-global.251d01.css";
-
```

Comment:
I'm not sure we can remove this wholesale because it will affect the other components in the sb - did you mean to remove this from your commit?

Reply:
Apologies @jeffbelser-okta , [I misunderstood our original interaction around this](https://github.com/okta/odyssey/pull/1160#issuecomment-963715860). I took it to mean that you were OK with us removing these styles instead of doing it locally on a one off basis.

I have no preference and have reverted the commit which removed these styles. Should be good to go!